lib/internal/process/promises.js

```diff
@@ -124,11 +126,18 @@ function promiseRejectHandler(type, promise, reason) {
   }
 }
 
+const multipleResolvesDeprecate = deprecate(
+  () => {},
+  'The multipleResolves event has been derpecated.',
+  'DEPXXXX'
+);
 function resolveError(type, promise, reason) {
   // We have to wrap this in a next tick. Otherwise the error could be caught by
   // the executed promise.
   process.nextTick(() => {
```

Comment:
Can't you do this instead?
```suggestion
  process.nextTick(multipleResolvesDeprecate);
```